src/App.jsx

```diff
@@ -10,6 +58,21 @@ function App() {
       <h1>Weather App</h1>
       <div className="card">
         {/* Follow the weather app instructions on the gitbook to implement this exercise */}
```

Comment:
form does not need the method or action here.
Those properties use the default action of the form